packages/compass-import-export/src/components/import-toast.tsx

```diff
@@ -98,14 +98,49 @@ export function showBloatedDocumentSignalToast({
         <Body as="span">
           The imported documents might exceed a reasonable size for performance.
         </Body>
-        <br />
-        <Body
-          as="strong"
-          onClick={onReviewDocumentsClick}
-          className={reviewDocumentsCTAStyles}
-        >
-          Review Documents
+        {onReviewDocumentsClick && (
+          <>
+            <br />
+            <Body
+              as="strong"
+              onClick={onReviewDocumentsClick}
+              className={reviewDocumentsCTAStyles}
+            >
+              Review Documents
+            </Body>
+          </>
+        )}
+      </>
+    ),
+    variant: 'note',
+  });
+}
+
+export function showUnboundArraySignalToast({
+  onReviewDocumentsClick,
+}: {
+  onReviewDocumentsClick?: () => void;
+}) {
+  openToast(bloatedDocumentSignalToastId, {
+    title: 'Unbounded array(s) detected',
```

Comment:
Should we match this toast title to the signal title i.e. - "Large array detected"?

Reply:
Just realised that the discussion in tech design was about the signal title, not sure if it applies to the toast as well.

Reply:
Hmmm, good catch, I'll double-check

Reply:
Yeah, looking closer at the designs the text there just wasn't updated in other places too, so make sense to just use the new title here too. Updated in https://github.com/mongodb-js/compass/pull/4688/commits/3dd36a55bd39bf943d6c61c1c8dea887b68b9ddf